src/cursor/cur_file.c

```diff
@@ -179,12 +179,17 @@ __curfile_search(WT_CURSOR *cursor)
 	WT_CURSOR_BTREE *cbt;
 	WT_DECL_RET;
 	WT_SESSION_IMPL *session;
+	uint64_t start, stop;
```

Comment:
There was a serious effort in WT-3790 to name all timings as `time_start` and `time_stop`. Please adjust these variables.  (This applies to the 4 or 5 other places this appears too)